routes/login/login.spec.js

```diff
@@ -80,7 +80,7 @@ describe('Test /login responses', () => {
     expect(response.statusCode).toBe(422)
   })
 
-  const codes = ['QWER1234', 'qwer1234']
+  const codes = ['A5G98S4K1']
```

Comment:
This should still work with lower-case letters. Had the extra test just to confirm the behaviour.